src/main/java/core/GraphReducer.java

```diff
@@ -313,6 +419,22 @@ public static Boolean collapseBubble(HashMap<Integer, Node> nodeMap, Node parent
     }
 
     /**
+     * General method to collapse a node into another node
+     * @param parent The node that will be retained
+     * @param child The node that will be collapsed
+     * @param zoomLevel The previous zoom level
+     */
+    public static void collapseNodeIntoParent(Node parent, Node child, int zoomLevel) {
+        parent.unionGenomes(child);
+        parent.setNucleotides(parent.getNucleotides() + child.getNucleotides());
```

Comment:
a variable called 'nucleotides' to represent the number of nucleotides is wrong. Variable name should be renamed to something like 'numNucleotides'